autoload/characterize.vim

```diff
@@ -1063,11 +1397,11 @@ let s:d[0x007C]='VERTICAL LINE'
 let s:d[0x007D]='RIGHT CURLY BRACKET'
 let s:d[0x007E]='TILDE'
 let s:d[0x007F]='DELETE'
-let s:d[0x0080]='PADDING CHARACTER'
-let s:d[0x0081]='HIGH OCTET PRESET'
+let s:d[0x0080]='<control>'
+let s:d[0x0081]='<control>'
```

Comment:
It's a loss of information for all of these to become `<control>`.
(I think I had to fix these up manually last time.)